.gitignore

```diff
@@ -19,15 +23,27 @@ npm-debug.log
 
 # User-specific files
```

Comment:
gotta check with Steph or Ian about changes in the .gitignore. I'm less clear on what's what, but they will know